plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -3071,7 +3071,6 @@ public void run(HiveMetastoreClosure metastore)
             boolean done = false;
             try {
                 metastore.createTable(identity, newTable, privileges);
-                done = true;
```

Comment:
if `done` is used within `catch` clause only, please move the declaration there.